### ISSUE TITLE
Which parameter in the function ants.build_template() in antspy corresponds to the -r option in buildtemplateparallel.sh

### ISSUE THREAD

**Issue by angolin22** [ANTsX/ANTsPy#513]
hello, thanks for your work! I want to ask some questions:
1. Could I ask that which parameter in the function `ants.build_template()` in antspy corresponds to the` -r `option in buildtemplateparallel.sh
2. Does CT images of constructing the template require preprocessing, such as resampling to isotropy and normalizing the Hu value to 0-255?
Thanks for your reply!

**Comment by ntustison** [comment]
> Could I ask that which parameter in the function ants.build_template() in antspy corresponds to the-roption in buildtemplateparallel.sh

I don't think it's available.  But you can easily get something close enough by rigidly registering all your data to one of your subjects or an existing template.

> Does  ants.build_template() support mask-based registration?

No.

> Does CT images of constructing the template require preprocessing, such as resampling to isotropy and normalizing the Hu value to 0-255?

Yeah, that's probably a good idea.

**Comment by angolin22** [comment]
thanks for your reply

**Comment by angolin22** [comment]
Hello, I want to ask about the parameters of the function `ants.build_template()`. Is there such a parameter `Gaussian regularization` in it.
Thanks for your reply.

**Comment by ntustison** [comment]
No.  FYI, you can find the parameters list from the help menu, ``help(ants.build_template)``.

Also, the python version is a more limited version of what's available in the bash scripts so don't expect equivalency in the set of available parameters.

**Comment by angolin22** [comment]
OK，Thanks.
 
Also,I want to ask some questions.
1.  I would like to ask if the algorithm in `ants.build_template` is the weighted average template algorithm based on kernel regression? 
2.  If a template is generated for several CT images of different sizes, what is the size of the final template?

**Comment by nimzodisaster** [comment]
I think the confusion is that the the documentation refers to **kwargs, but doesn't indicate what those might be, potentially suggesting expert, but hidden options.
